### ISSUE TITLE
Add metadata for people

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBOFoundry.github.io#24]
Each person (OBO team, or possibly also ontology developers) could have an entry in the config.yml file

E.g.

```
team:
  - name: Chris Mungall
    orcid: ORCIDURL
    github: cmungall
    roles: [twg,...]
```

Or alternatively, one md per person (that can be collected same way as ontologies)

This would be easy to render as html in jekyll

The YAML would have a json-ld context allowing direct translation to RDF

**Comment by cmungall** [comment]
I just discovered this on the obo-ops google drive: https://docs.google.com/spreadsheets/d/1aLkZyVryQaa5BmDg8fG650-P2bhaoqZT6OmsNLKPQGc/edit#gid=0

**Comment by mcourtot** [comment]
I like the idea of now md per person - which opens the door for self-edit. the ontology-forum used to have a page like that as well. I'll try and see if I can play around and duplicate/update the ontology details mechanism for people.

**Comment by mcourtot** [comment]
I have started trying this out (should probably create a branch for the changes) with currently one md per person.
I'm not sure how to link people to ontologies. 
(1) there are ontologies with no people (but instead for example a mailing-list)
(2) there are people working on multiple ontologies
(3) there will be ontologies with multiple people

I'm wondering if we should have an extra entry in ontologies which would be "people" - but then we'd need to parse that to get a list of distance people across all ontologies and create an md for them when a new one is added to an ontology.

In short, it'd be nice to have cross reference ontology -> people and people -> ontology, with a many to may relation, but I'm not quite sure how to do that. Suggestions?

**Comment by mcourtot** [comment]
created branch issue 24. index2.html calls the people_table, the 2 md files under the people directory drive the content (need to run make first)

**Comment by mcourtot** [comment]
ok. thinking about it and in light of our discussion I think:
- we should have a people registry (and not conflate roles/try to parse stuff)
- each person gets an URI
- ontologies mds use those URIs

**Comment by cmungall** [comment]
@mcourtot do we have a plan here or should we de-prioritize and close this?

**Comment by nlharris** [comment]
> @mcourtot do we have a plan here or should we de-prioritize and close this?

sounds like the latter?

**Comment by nlharris** [comment]
can this be closed?

**Comment by matentzn** [comment]
Closing now for inactivity.
